### DISCUSSION TITLE
Unable to figure out why my container keeps getting a 172 address, can I borrow a second pair of eyes?

### DISCUSSION THREAD

**Discussion by RiffyDivine** [louislam/dockge#404] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Now I never had this issue before when I was using ubuntu but since moving to arch I am now having an issue with my arr stack all getting internal 172 network address over just bridging or hosting to the systems ip. So I pulled one from it and tested with that one as a new container and same issue. Can someone look at the compose file and let me know where my error is?
I am at a bit of a loss for where the issue is coming from. It is leading to issues since the link inside dockge is for my local ip and port, when setting them up it errors out cause they all want the internal ip address docker is giving them and it's just leading to issue. 

version: "3.7"
services:
  sabnzbd:
    image: lscr.io/linuxserver/sabnzbd:latest
    container_name: sabnzbd
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=America/Chicago
    volumes:
      - /docker/appdata/sabnzbd/config:/config
      - /incoming:/downloads #optional
      - /incoming/incomplete:/incomplete-downloads #optional
    ports:
      - 8080:8080
    restart: unless-stopped
networks: {}

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

arch

### 🌐 Browser

firefox

### 🐋 Docker Version

24.0.7

**Comment by trythatagain** [comment]
I use OMV which is Debian based, so I've not used Ubuntu (which I know is based on Debian). If I want my docker network to bridge to my main network, I have to create a docker bridge network.

1 other thought occurred to me, have you set 'Primary Hostname' from the settings menu? Just a thought, not trying how to suck eggs :)

**Reply by RiffyDivine** [reply]
Sorry been busy. But I switched back to ubuntu and now I am having the issue on it also. Yes I did set the primary hostname. My other containers are fine it's just anything that needs to talk to other ones is having issues.
